libs/render_delegate/reader.cpp

```diff
@@ -292,7 +294,7 @@ void HydraArnoldReader::ReadStage(UsdStageRefPtr stage,
         ChooseRenderSettings(stage, _renderSettings, _time);
         if (!_renderSettings.empty()) {
             UsdPrim renderSettingsPrim = stage->GetPrimAtPath(SdfPath(_renderSettings));
-            ReadRenderSettings(renderSettingsPrim, arnoldRenderDelegate->GetAPIAdapter(), _time, _universe, renderCameraPath);
+            ReadRenderSettings(renderSettingsPrim, arnoldRenderDelegate->GetAPIAdapter(), this, _time, _universe, renderCameraPath);
```

Comment:
instead of changing ReadRenderSettings which is in charge of reading a RenderSettings prim, could we instead create a function ReadCommandLine() that we call after ReadRenderSettings ? this way it will override whatever was set in the render settings